### DISCUSSION TITLE
Trouble Connecting to Supabase from Flutter

### DISCUSSION THREAD

**Discussion by boldrhino** [supabase/supabase#5867] [General]
Supabase Community,

This might be a very stupid question......

I am testing connectivity to Supabase from Flutter, simply trying to pull back a list of table rows. However, it doesn't look like my app is connecting using the standard Supabase URL and Key. I've not setup any authentication as yet, will do that once I know I can connect and get data.

Do I need to authenticate BEFORE Supabase will allow me to pull back database rows? Or will my URL and KEY suffice?..and therefore I am missing something

final client = SupabaseClient(supabaseUrl, supabaseKey);
final response = await client.from(table).select().execute();

Any help greatly appreciated!

**Comment by GaryAustin1** [comment]
I've not used flutter, but don't know why it would act different in this case.  In general as long as the client has url and key set you can access data... IF rls is off for that table, or the policy is set to allow anon users.
